data/EnemyData.tsv

```diff
@@ -118,13 +118,13 @@ enemy_index	name	max_hp	atk	pdef	mdef	drop_chance	drop_type	drop_item
 0x74	Crazy Horse\x00	64	10	2	40	5	item	\u874fPotion
 0x75	Nightmare\x00	200	30	24	100	0	None	None
 0x76	Death Machine\x00	2000	128	80	200	5	armor	\u8748Genji Armor
-0x77	Lich\x00	1200	40	40	120	2	item	\u874fDry Ether
+0x77	Lich\x00	1200	40	40	120	100	armor	\u874bRibbon

```

Comment:
So I don't mind that you made this change, but it isn't called out in the site (The text on the flag suggests the fiends only drop Ribbons with the new item flag on, but this looks like it will happen regardless)

Reply:
Fair - what I'll do is I'll have a duplicated tsv like the other ones.